source/puppet/5.3/hiera_automatic.md

```diff
@@ -6,20 +6,20 @@ title: "Looking up data with Hiera"
 [data_type]: ./lang_data_type.html
 [editing_data]: ./hiera_merging.html
 
-{:.Concept}
-## Class parameters 
-  
+{:.concept}
+## Class parameters
+
 Puppet looks up the values for class parameters in Hiera, using the fully-qualified name of the parameter (`myclass::parameter_one`) as a lookup key.
 
 Most classes need configuration, and you can specify them as parameters to a class as this will lookup the needed data if not directly given when the class is included in a catalog. There are several ways Puppet sets values for class parameters, in this order:
 
-1. If you’re doing a resource-like declaration, Puppet uses parameters that are explicitly set (if explicitly setting undef, a looked up value or default will be used).
+1.  If you're doing a resource-like declaration, Puppet uses parameters that are explicitly set (if explicitly setting undef, a looked up value or default will be used).
```

Comment:
Consistently indenting Markdown lists is difficult, and a mistake can be hard to spot. One way to keep lists consistent is to use a consistent number of characters to indent all list items.

I've changed all ordered and unordered lists to use a consistent four-character indent. In this instance, I added a space as the fourth character (`1`, `.`, space, **space**).